client/app/private-database/configuration/private-database-configuration.html

```diff
@@ -1,76 +1,67 @@
 <div class="container-fluid px-0" data-ng-controller="PrivateDatabaseConfigurationsCtrl as configurationsCtrl">
 
+    <div data-ovh-alert="{{alerts.main}}"></div>
+
     <div class="text-center" data-ng-if="configurationsCtrl.loading">
         <oui-spinner data-size="l"></oui-spinner>
     </div>
 
-    <div class="row" data-ng-if="!configurationsCtrl.loading">
-        <div class="col-md-9">
-            <div data-ovh-alert="{{alerts.main}}"></div>
+    <div data-ng-if="!configurationsCtrl.loading">
+
+        <div class="alert alert-info" role="alert"
+             data-i18n-static="privateDatabase_configurations_general_postgresql_soon"
+             data-ng-if="configurationsCtrl.database.type === 'postgresql'">
+        </div>
+        <div class="alert alert-info" role="alert"
+             data-i18n-static="privateDatabase_capability_disabled_explanation"
+             data-ng-if="configurationsCtrl.database.type !== 'postgresql' && configurationsCtrl.configurations.length === 0">
```

Comment:
Use one time binding every time it's possible.
Several places in this file

Reply:
I cannot use single binding there, because data is not loaded at controller instantiation

Reply:
ok! but there is other examples: `text="{{ tr('privateDatabase_configuration_confirm_btn') }}"`..